### PR TITLE
Adding mention of PnP PowerShell to enable sensitivity labels

### PR DESCRIPTION
**Pull request by KoenZomers**
Adding mention of PnP PowerShell cmdlet to enable sensitivity labels. The EnableAIPIntegration parameter referred to will be added in the PnP PowerShell release of  May 8th, 2020.

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/sensitivity-labels-sharepoint-onedrive-files.md
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitivity-labels-sharepoint-onedrive-files.md]:
```suggestion
As an alternative to using the compliance center, you can enable support for sensitivity labels by using the [Set-SPOTenant](https://docs.microsoft.com/powershell/module/sharepoint-online/set-spotenant?view=sharepoint-ps) cmdlet from SharePoint Online PowerShell or the [Set-PnPTenant](https://docs.microsoft.com/powershell/module/sharepoint-pnp/set-pnptenant?view=sharepoint-ps) cmdlet passing in the _EnableAIPIntegration_ parameter from PnP PowerShell. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@cabailey - PR has been copy edited and is ready for final review. 

Thank you.

cc: @kenwith

**Comment by cabailey** [comment]
Looking into this - please don't merge for now.

**Comment by KoenZomers** [comment]
Please note that support for `Set-PnPTenant -EnableAIPIntegration` will be released next Tuesday, June 9th, 2020 in PnP PowerShell. So at this moment it's not available yet in the current release.

**Comment by cabailey** [comment]
@KoenZomers  Thank you for your suggestion but after consideration, we've decided not to accept the change for this page.  In context it introduces an alternative that adds complexity for customers that can hinder more than help.  It's missing information such as how to install this module, how to authenticate, and refers to the parameter that isn't yet mentioned. We could add all that, but then we would have to also add guidance which method to use, and why.  

Many configurations have valid alternative methods - for example, double-clicking vs right-clicking vs selecting a menu option or button.  Instead of documenting them all, the writing guidance is to choose one to minimize complexity for customers.  Perhaps there is business value for the target audience to use this method over the currently documented method?  Happy to continue this discussion on my Microsoft email account.

**Comment by KoenZomers** [comment]
@cabailey, fair enough

**Pull request closed without merging**
